trunk/bin/lscloop.py

```diff
@@ -186,8 +184,7 @@ def fpack(filename):
             elif args.stage == 'getmag':  # get final magnitude from mysql
                 lsc.myloopdef.run_getmag(ll['filename'], args.output, args.interactive, args.show, args.combine, args.type, args.uploadtosnex2)
             elif args.stage == 'psf':
-                catalogue = lsc.util.getcatalog(args.name, args.field) if args.field else args.catalogue
-                lsc.myloopdef.run_psf(ll['filename'], args.threshold, args.interactive, args.fwhm, args.show, args.force, args.fix, catalogue,
+                lsc.myloopdef.run_psf(ll['filename'], args.threshold, args.interactive, args.fwhm, args.show, args.force, args.fix, args.catalogue,
```

Comment:
Why do we no longer have to run getcatalog here?

Reply:
The issue is that, when you run `-s psf` with `--standard`, you were previously getting a catalog that belonged to the supernova, not the individual standard fields. So `getcatalog` needs to be handled on a per-image basis.